src/main/java/org/spongepowered/api/service/context/Context.java

```diff
@@ -44,18 +45,32 @@
     public static final String LOCAL_IP_KEY = "localip";
     public static final String LOCAL_PORT_KEY = "localport"; // This portkey is an old boot
 
-    private final Map.Entry<String, String> wrapped;
+    private final String key;
+    private final String value;
```

Comment:
How about renaming the fields to type and name, as this is their actual purpose. The key and value stuff is just for Map.Entry compatibility.